src/pages/home/HomePage.js

```diff
@@ -101,8 +102,8 @@ class HomePage extends PureComponent {
                     ROUTES.HOME,
                     ROUTES.SETTINGS,
                     ROUTES.NEW_GROUP,
-                    ROUTES.NEW_CHAT,
-                    ROUTES.SEARCH,
+                    ROUTES.IOU_REQUEST_MONEY,
+                    ROUTES.IOU_BILL_SPLIT,
```

Comment:
I think these should be added to the array and not replace the ones for new chat and search?

Reply:
Ah whoops, accidentally committed part of a temp git diff.